updater/internal/file/file.go

```diff
@@ -40,19 +41,29 @@ func CopyFile(logger *zap.Logger, pathIn, pathOut string, overwrite bool) error
 		}
 	}()
 
+	pathOutClean := filepath.Clean(pathOut)
+	var fileMode fs.FileMode
+	fileMode = 0600
```

Comment:
Can we condense this into a single line? 

```go
var fileMode fs.FileMode = 0600
```

or 

```go
fileMode := fs.FileMode(0600)
```

Reply:
@cpheps yep that would definitely be better. i meant to do this, but i must have forgot

Reply:
fixed